Add integration framework (#1132)
camunda/camunda-platform-docs: docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

---
id: connector-sdk
title: Connector SDK
description: Introduction to the Connector SDK
---

:::note

> The **connector SDK** is a prototype and subject to breaking changes. Use at your own risk!

Reviewer: ```suggestion
The **connector SDK** is in developer preview and subject to breaking changes. Use at your own risk!
```
